src/sedcli_main.c

```diff
@@ -493,9 +500,26 @@ static void print_blocksid_feat(struct sed_blocksid_supported_feat *blocksid)
 			blocksid->hardware_reset ? 1 : 0);
 }
 
+static void print_cnl_feat(struct sed_cnl_feat *cnl)
+{
+	sedcli_printf(LOG_INFO, "\nSED CNL FEATURES SUPPORTED\n");
+	sedcli_printf(LOG_INFO, "---------------------------\n");
+	sedcli_printf(LOG_INFO, "\tNamespace Non-Global Range Locking objects Supported : %s\n",
+			cnl->ranges_rsvd.range_c ? "Y" : "N");
+	sedcli_printf(LOG_INFO, "\tLocking table contains %s Namespace Non-Global Range Locking objects\n",
```

Comment:
please stick to existing convention when displaying discovery data. my recommendation is to print it in follwoing way:
"Namespace Non-Global Range Locking object:    ONE OR MORE"

Reply:
Will do it